### PR TITLE
Docs/jira server plugin

### PR DESCRIPTION
**Pull request by chriswk**

### CODE REVIEW

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+![Jira Manage Apps - Unleash Admin](/img/jira_server_manage_unleash_admin.png)
+
+Following the "Configure servers" link will take you to a config page where you can setup your connection(s) to Unleash
+servers. The Unleash Jira server plugin supports having a global server configured for all Jira projects, or a specific
+server configured for a specific Jira project.
+
+![Jira Manage Apps - Unleash Server Admin](/img/jira_server_manage_servers.png)
+
+Once you have configured at least one Unleash server, your users should be ready to [Use the Jira Server plugin](/integrations/jira_server_plugin_usage)
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:68]:
```suggestion
Once you have configured at least one Unleash server, your users should be ready to [use the Jira Server plugin](/integrations/jira_server_plugin_usage)
```

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+With the Unleash Jira server plugin you can create new feature toggles in Unleash from a Jira issue as well as link
+existing toggles to Jira issues.
+
+The plugin also shows you current status of connected toggles.
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash Instance admin to configure an admin API token.
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:17]: Should 'Instance' be capitalized?

```suggestion
You will need an Unleash instance admin to configure an admin API token.
```

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+servers. The Unleash Jira server plugin supports having a global server configured for all Jira projects, or a specific
+server configured for a specific Jira project.
+
+![Jira Manage Apps - Unleash Server Admin](/img/jira_server_manage_servers.png)
+
+Once you have configured at least one Unleash server, your users should be ready to [Use the Jira Server plugin](/integrations/jira_server_plugin_usage)
+
+### Edit existing servers
+
+In the list of known servers, each server has a connected edit button. Clicking the Edit button brings up the edit
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:72]:
```suggestion
In the list of known servers, each server has a connected edit button. Clicking the edit button brings up the edit
```

File: website/docs/integrations/jira-server-plugin-usage.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+
+### Disconnecting toggle from Issue
+
+If a toggle is no longer relevant for your Jira Issue, you can disconnect it using the Disconnect toggle button (
+provided your user has edit rights on the issue)
+
+![Jira Server - Disconnect toggle](/img/jira_server_disconnect_toggle.png)
+
+Once you click the button, you'll need to confirm the dialog that opens up
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-usage.md:45]:
```suggestion
Once you click the button, you'll need to confirm the dialog that opens up.
```

File: website/docs/integrations/jira-server-plugin-usage.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+![Jira Server - Disconnect toggle dialog](/img/jira_server_disconnect_toggle_dialog.png)
+
+if confirmed, the toggle will be disconnected immediately. However, the plugin will not delete the toggle from Unleash,
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-usage.md:49]:
```suggestion
If confirmed, the toggle will be disconnected immediately. However, the plugin will not delete the toggle from Unleash,
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
This looks great! Just a few rogue capital letters that we might want to address before pushing it out. Great work! 🎉

File: website/docs/integrations/jira-server-plugin-installation.md (hunk 1 above)
Comment (sighphyre) [reply website/docs/integrations/jira-server-plugin-installation.md]: Good eyes, thank you!

File: website/docs/integrations/jira-server-plugin-installation.md (hunk 2 above)
Comment (sighphyre) [reply website/docs/integrations/jira-server-plugin-installation.md]: Actually reads better without it at all - I'm yanking it out

**Pull request merged**